starlette/middleware/errors.py

```diff
@@ -137,7 +137,9 @@ class ServerErrorMiddleware:
     def __init__(
         self,
         app: ASGIApp,
-        handler: typing.Optional[typing.Callable] = None,
+        handler: typing.Optional[
+            typing.Callable[[Request, Exception], typing.Any]
+        ] = None,
```

Comment:
Isn't this supposed to be `ExceptionHandler` as well? It's a question, not a suggestion.

Reply:
Yes it is, this was discussed earlier: https://github.com/encode/starlette/pull/2180#discussion_r1225486748. We could keep it as `Any`, or use `Union[Reponse, Awaitable[Response]]` and then add some casts or `type: ignore`